### PR TITLE
Modify reconciler to accept replicaId env var

### PR DESCRIPTION
**Pull request by connoravo**
In order for us to verify if Hbase has the data stored to the storage medium, we must read from a replica. This will confirm if the information is stored on the storage medium and not being returned from the masters memstore.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -30,6 +30,7 @@ services:
       HBASE_CLIENT_SCANNER_TIMEOUT_PERIOD_MS: "60000"
       HBASE_PAUSE_MILLISECONDS: "50"
       HBASE_RETRIES": "3"
+      HBASE_REPLICA_ID: "2"
```
Comment (dan-hill2802) [inline docker-compose.yaml:33]: does this need copying over to the infra too?

Comment (connoravo) [reply docker-compose.yaml:33]: It will do ye. I haven't even linked this in the PR channel yet ha!

Comment (steveburton4) [reply docker-compose.yaml:33]: It has a default though so can be released without it

File: src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/repositories/impl/HBaseRepositoryImpl.kt [hunk 2]
```diff
@@ -70,10 +82,13 @@ class HBaseRepositoryImpl(private val connection: Connection, private val tableN
 
     private var _mutableMap: MutableMap<String, Boolean> = mutableMapOf()
 
-    private fun get(id: String, version: Long) = Get(tableNameUtil.decodePrintable(id)).apply {
+    private fun get(id: String, version: Long, replicaId: Int) = Get(tableNameUtil.decodePrintable(id)).apply {
         setTimeStamp(version)
         isCheckExistenceOnly = true
         consistency = Consistency.TIMELINE
+        if (replicaId > 0) {
```
Comment (steveburton4) [inline src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/repositories/impl/HBaseRepositoryImpl.kt]: Needs to be >=

Comment (steveburton4) [reply src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/repositories/impl/HBaseRepositoryImpl.kt]: I would add a unit test specifically checking passing in 0 and using 0 to ensure that is caught.

**Review by steveburton4 — CHANGES_REQUESTED** [review]

File: src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/repositories/impl/HBaseRepositoryImpl.kt (hunk 2 above)
Comment (connoravo) [reply src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/repositories/impl/HBaseRepositoryImpl.kt]: Check recent push please

**Review by steveburton4 — APPROVED** [review]

**Pull request merged**